react/components/LowStock.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment, FunctionComponent } from 'react'
+import classNames from 'classnames'
+import styles from '../styles.css'
+
+interface Props {
+  text: string
+  availableQuantity: number
+}
+
+const LowStock: FunctionComponent<Props> = ({ text, availableQuantity }) => {
+  const [before, after] = text.split('{quantity}')
```

Comment:
perhaps check if `text` is actually a string before splitting it

Reply:
`if (isLowStock && lowStockMessage) `

already guarantees text is a string